src/transformers/integrations.py

```diff
@@ -932,41 +932,38 @@ def __del__(self):
 
 
 class NeptuneCallback(TrainerCallback):
-    """
+    """TrainerCallback that sends the logs to [Neptune](https://neptune.ai).
     TODO: NPT-12189 - Update the docstring
-    A [`TrainerCallback`] that sends the logs to [Neptune](https://neptune.ai).
 
     Args:
-        api_token ('str', *optional*):
-            Neptune API token obtained from <https://neptune.ai> upon registration.
-            It is recommended to keep it in the `NEPTUNE_API_TOKEN` environment variable
-            -- in this case it does not have to be provided here.
-            Read more in the
-            `Neptune installation docs <https://docs.neptune.ai/getting-started/installation>`_.
-        project (`str`, *optional*):
-            Name of the project. Format: "WORKSPACE/PROJECT".
-            If `None` (default), the value of `NEPTUNE_PROJECT` environment variable will be used.
-            You need to create the project in <https://neptune.ai> first.
-        name (`str`, *optional*):
-            Name of the run, it appears in the "all metadata/sys" section in Neptune UI.
-        base_namespace (`str`, *optional*, defaults to "finetuning"):
-            Root namespace within Neptune's run.
-        log_parameters (`bool`, *optional*, defaults to True):
-            If True, log all trainer arguments and model parameters provided by the trainer.
-        log_checkpoints (`str`, *optional*, defaults to "best"):
+        api_token (`str`, optional):
+            Neptune API token obtained upon registration. You can leave this argument out if you have
+            saved your token to the `NEPTUNE_API_TOKEN` environment variable (strongly recommended).
+            See full setup instructions in the [docs](https://docs.neptune.ai/getting-started/installation).
+        project (`str`, optional):
+            Name of an existing Neptune project, in the form: "workspace-name/project-name".
+            You can find and copy the name from the project Settings -> Properties in Neptune.
+            If None (default), the value of the `NEPTUNE_PROJECT` environment variable will be used.
+        name (`str`, optional):
+            Custom name for the run. It is shown in the Neptune app and stored in the "sys/name" field.
+        base_namespace (`str`, optional, defaults to "finetuning"):
+            Root namespace within the Neptune run.
+        log_parameters (`bool`, optional, defaults to True):
+            If True, logs all trainer arguments and model parameters provided by the trainer.
+        log_checkpoints (`str`, optional, defaults to "best"):
             If "same", upload checkpoints whenever they are saved by the trainer.
```

Comment:
The docstrings weren't prepared by me and I didn't check them before. There is another option `"last"` which means it will store in Neptune only the last saved one (similar to "same" option but only the most recent will be available in Neptune)